R/utils_accessibility.R

```diff
@@ -126,53 +126,65 @@ nhs_card_tabstop <- function(header, tabindex = 0, ...) {
 }
 
 
-
-
-
-
-
 #' accessible_radio_button
 #'
-#' @param ... radio content
+#' @inheritParams shiny::radioButtons
 #'
-#' @noRd
+#' @return HTML
+#' @export
 #'
+#' @examples
+#' accessible_radio_button("rb", "Choose one:",
+#'                         choiceNames = list(
+#'                           "apple",
+#'                           "banana",
+#'                           "cherry"
+#'                         ),
+#'                         choiceValues = list(
+#'                           "A", "B", "C"
+#'                         ))
+# Begin Exclude Linting
```

Comment:
Normally we want to use snake_case for function and variable names. In this case we are basing the function off an existing one that uses camelCase, so makes sense to keep the original naming convention for clarity. Need to exclude linting then. Because the args are spread over several lines, easier to exclude the whole block, rather than individual lines.

Reply:
yes, i will add to lintr to ignore camelCase.